explore-the-admin-ui.md

```diff
@@ -4,4 +4,10 @@ toc: false
 feedback: false
 ---
 
-Docs coming soon. For guidance on accessing the Admin UI, see [Start a Local Cluster](start-a-local-cluster.html) and [Manual Deployment](manual-deployment.html).
\ No newline at end of file
+CockroachDB's Admin UI provides detail about your cluster's performance and health.
+
+You can access the CockroachDB Admin UI from any node in a cluster on port `8080` via HTTP (e.g., `http://<node IP address>:8080`).
```

Comment:
This is only true for the first node if you accept the `--http-port` default. Normally, you can access the admin ui from any node on its whatever port was set as `--http-port`. Can you rework this a bit?